web-api/src/main/java/com/graphhopper/util/CustomModel.java

```diff
@@ -120,6 +120,10 @@ public CustomModel setDistanceInfluence(Double distanceFactor) {
         return this;
     }
 
+    /**
+     * @return the distance influence of this CustomModel. 0 is returned if it wasn't set (i.e. it was null)
+     * @see #hasDistanceInfluence()
+     */
```

Comment:
Can we not just return `null`? `null` means it's not there, so exactly what we want?

Reply:
Did you commit this somewhere? Where was there a problem? I would prefer to keep it as `null` in case it is missing. The special value `0` might even be needed for some cases.